openssf-charter.md

```diff
@@ -0,0 +1,447 @@
+  <li>The Linux Foundation's Good Standing Policy is available at <https://www.linuxfoundation.org/good-standing-policy> and will apply to Members of the OpenSSF.</li></ol>
+
+10. **Trademarks**
+<ol type="a">
+  <li>
+
+  Any trademarks relating to the OpenSSF or a Technical Initiative, including without limitation any mark relating to any Conformance Program, must be transferred to and held by the Linux Foundation or one of its affiliates and available for use pursuant to the trademark usage policy of the Linux Foundation (available at [https://www.linuxfoundation.org/trademark-usage](https://www.linuxfoundation.org/trademark-usage) or such affiliate.</li></ol>
+
+11. **Antitrust Guidelines**
+<ol type="a">
+  <li>
+  
+  All Members must abide by The Linux Foundation's Antitrust Policy available at [http://www.linuxfoundation.org/antitrust-policy](http://www.linuxfoundation.org/antitrust-policy). </li>
+    <li>All Members must encourage open participation from any  organization able to meet the membership requirements, regardless of competitive interests. Put another way, the Governing Board will not seek to exclude any member based on any criteria, requirements or reasons other than those that are reasonable and applied on a non-discriminatory basis to all members.</li></ol>
+
+12. **Budget**
+<ol type="a">
+  <li>If the OpenSSF decides to raise funds, the Governing Board will
+        approve an annual budget and never commit to spend in excess of funds raised. The budget and the purposes to which it is applied must be consistent with both (a) the non-profit and tax-exempt mission of the Linux Foundation and (b) the aggregate goals of the Technical Initiatives.</li>
+        </br>
+  <li>The Linux Foundation will provide the Governing Board with regular reports of spend levels against the budget. Under no circumstances will the Linux Foundation have any expectation or obligation to undertake an action on behalf of the OpenSSF or otherwise related to the OpenSSF that is not covered in full by funds raised by the OpenSSF.</li>
+        </br>
+  <li>In the event an unbudgeted or otherwise unfunded obligation arises related to the OpenSSF, the Linux Foundation will coordinate with the Governing Board to address gap funding requirements.</li></ol>
+
+13. **General & Administrative Expenses**
+<ol type="a">
+  <li>The Linux Foundation will have custody of and final authority over the usage of any fees, funds and other cash receipts.</li>
+        </br>
+  <li>A General & Administrative (G&A) fee will be applied by the Linux Foundation to funds raised to cover membership records, finance, accounting, and human resources operations. The G&A fee will be 9% of the OpenSSF's first \$1,000,000 of gross receipts each year and 6% of the OpenSSF's gross receipts each year over \$1,000,000. Individual Technical Initiative funding arrangements may be setup under alternative arrangements by approval of the Governing Board and the Linux Foundation.</li></ol>
```

Comment:
For some reason these "\"s render. See if you can get by without the escapes in front of the $s.